bundle/edu.gemini.catalog/src/main/java/jsky/catalog/TablePlotSymbol.java

```diff
@@ -569,6 +533,7 @@ public String getRatio() {
     }
 
     /** Return the symbol's ratio, evaluating the expression for the given row. */
+    @SuppressWarnings("EmptyCatchBlock")
```

Comment:
I'm happy to go through and make whatever changes people think should be done.
Logging the errors?